modules/ROOT/pages/csrf-security.adoc

```diff
@@ -58,7 +58,7 @@ To disable the CSRF protection, edit the configuration file and change the `secu
 You can set the secure flag on the `X-Bonita-API-Token` cookie to make sure the browser will only transmit the cookie if the page is HTTPS.
 
 Activating the addition of the secure flag can be done in the file `security-config.properties` for the whole platform.
-The default version of this file is located in `setup/platform_conf/initial/platform_portal`. In order to change the configuration on an installation whose platform has already been initialized, use the xref:bonita-bpm-platform-setup.adoc[platform setup tool] to retrieve the current configuration and update the file in `setup/platform_conf/current/platform_portal`. Then use the tool again to save your changes into to the database.
+The default version of this file is located in `setup/platform_conf/initial/platform_portal`. In order to change the configuration on an installation whose platform has already been initialized, use the xref:bonita-platform-setup.adoc[platform setup tool] to retrieve the current configuration and update the file in `setup/platform_conf/current/platform_portal`. Then use the tool again to save your changes into to the database.
```

Comment:
setup/platform_conf/initial/platform_portal` => shouldnt we change the name? there is portal in it.

Reply:
Same as above